src/components/formItem/formItem.svelte

```diff
@@ -240,6 +240,8 @@
     background: var(--background);
     box-shadow: var(--shadow);
     border: 1px solid var(--border-color);
+    transition: box-shadow 0.15s ease-in-out, border-color 0.15s ease-in-out;
```

Comment:
I think in other places we use 0.12s or 0.2s depending on the animation. Probably not super important though

Reply:
Might be worth creating a token for transitions.

Reply:
Nice, thanks, updated to 0.2